package.json

```diff
@@ -29,8 +29,9 @@
   },
   "dependencies": {
     "dotenv": "^16.0.1",
-    "mixpanel-browser": "^2.43.0",
+    "ember-cli-app-version": "^5.0.0",
```

Comment:
I think we should have a separate PR to update the CLI to separate the concerns. Is that possible?